src/ScottPlot4/ScottPlot.Demo/ScottPlot.Demo.Avalonia/CookbookControl.axaml

```diff
@@ -13,6 +13,7 @@
 		<Grid.RowDefinitions>
 			<RowDefinition Height="auto"/>
 			<RowDefinition Height="*"/>
+            <RowDefinition Height="auto"/>
```

Comment:
Formatting

Reply:
Uuuh, some ugly tabs are used all around here ... Will be fixed :D